woocommerce/woocommerce
# send additional properties with Marketplace Suggestions tracks events

### All Submissions:


### Changes proposed in this Pull Request:
Adds some additional context info to Marketplace Suggestions tracks events.

### How to test the changes in this Pull Request:
- This PR depends on changes to the suggestion data from WCCOM – this PR is in progress (`#5738`). This PR should be tested with the updated data.
- Checkout this branch and get it set up as the WooCommerce plugin in your development environment. You may need to delete cookies, reset dismissed suggestions, and update the cached suggestion data.
- Log in as a store administrator and view suggestions & interact with suggestions – click the call-to-action, dismiss suggestions.
- Tracks events should have appropriate values for `context target product promoted`.
